doc/cascadia/Unittesting-CppWinRT-Xaml.md

```diff
@@ -12,7 +12,7 @@ were used in the application, and components that used XAML UI elements.
 
 ## Prerequisites
 
-Make sure you're using at least the 2.0.190605.7 version of the CppWinRT nuget
+Make sure you're using at least the 2.0.190730.2 version of the CppWinRT nuget

```

Comment:
Actually I don't think this one is needed.

Reply:
```suggestion
Make sure you're using at least the 2.0.190605.7 version of the CppWinRT nuget
```